model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/events/ClientRemovedEvent.java

```diff
@@ -45,8 +47,11 @@
     private String realmId;
     // roleId -> roleName
     private Map<String, String> clientRoles;
+    private static final Logger log = Logger.getLogger(ClientRemovedEvent.class);
```

Comment:
Is it a bit strange to have tracing only for client-removed? Would we want to either have tracing for everything, or nothing? Or, maybe having it for one thing let's us at least validate that things are working, without the effort of implementing logging for all "events".

Reply:
I'd rather keep this as limited as possible to keep the event objects as small as possible in memory, and in doing so I prefer the approach chosen in https://github.com/stianst/keycloak/blob/ec09ebac93c9e2491845ad4b26888bb558069b89/model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/events/ClientRemovedEvent.java - to pick a single event. I only lowered the level from info to trace.

Reply:
Yeah, I think that's a good trade-of, we really only need a single event to verify/debug/experiment with this stuff